hedera-node/hedera-smart-contract-service-impl/src/test/java/com/hedera/node/app/service/contract/impl/test/exec/TransactionProcessorTest.java

```diff
@@ -25,12 +25,12 @@
 import static org.mockito.Mockito.*;
 
 import com.hedera.hapi.node.base.ResponseCodeEnum;
```

Comment:
This class fails to hit `TransactionProcessor.java` L202 (`return new InvolvedParties(sender, relayer, pbjToBesuAddress(alias));` which might be an interesting lazy create thing to test?